pkg/api/norman/customization/catalog/store.go

```diff
@@ -52,15 +53,16 @@ func GetTemplateStore(ctx context.Context, managementContext *config.ScaledConte
 func (t *templateStore) extractVersionLinks(apiContext *types.APIContext, resource map[string]interface{}) map[string]interface{} {
 	schema := apiContext.Schemas.Schema(&managementschema.Version, client.TemplateVersionType)
 	r := map[string]interface{}{}
-	versionMap, ok := resource[client.CatalogTemplateFieldVersions].([]interface{})
+	versionMaps, ok := resource[client.CatalogTemplateFieldVersions].([]interface{})
 	if ok {
-		for _, version := range versionMap {
+		for _, version := range versionMaps {
 			revision := ""
 			if v, ok := version.(map[string]interface{})["revision"].(int64); ok {
 				revision = strconv.FormatInt(v, 10)
 			}
 			versionString, ok := version.(map[string]interface{})["version"].(string)
 			if !ok {
+				logrus.Trace("[templateStore] failed version type assertion for CatalogTemplateFieldVersion")
```

Comment:
I think this is bit confusing in its wording since "version" is a bit general. Perhaps...
```go
logrus.Trace("[templateStore] failed type assertion for field \"version\" in CatalogTemplateFieldVersion object")
```